commands/src/main/java/org/wildfly/extras/creaper/commands/elytron/AbstractAddCustom.java

```diff
@@ -0,0 +1,128 @@
+package org.wildfly.extras.creaper.commands.elytron;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.GroovyXmlTransform;
+import org.wildfly.extras.creaper.commands.foundation.offline.xml.Subtree;
+import org.wildfly.extras.creaper.core.ServerVersion;
+import org.wildfly.extras.creaper.core.offline.OfflineCommand;
+import org.wildfly.extras.creaper.core.offline.OfflineCommandContext;
+import org.wildfly.extras.creaper.core.online.OnlineCommand;
+import org.wildfly.extras.creaper.core.online.OnlineCommandContext;
+import org.wildfly.extras.creaper.core.online.operations.Address;
+import org.wildfly.extras.creaper.core.online.operations.Operations;
+import org.wildfly.extras.creaper.core.online.operations.Values;
+import org.wildfly.extras.creaper.core.online.operations.admin.Administration;
+
+public abstract class AbstractAddCustom implements OnlineCommand, OfflineCommand {
```

Comment:
Does it need to be `public`?

Reply:
This is extended by classes in different packages. I believe that DesignPatterns:FriendPackages can be use somehow, but I do not think it would be helpful here. Moreover it can be extended by some Creaper command for any Elytron custom resources in any different project in the future.

Reply:
Yea, sure, it's fine like this.